packages/node_modules/@webex/internal-plugin-ediscovery/test/integration/spec/ediscovery.js

```diff
@@ -0,0 +1,5 @@
+import '@webex/internal-plugin-ediscovery';
+
+describe('internal-plugin-ediscovery', () => {
+  it('do nothing', () => {});
```

Comment:
This is a step in the right direction!
Is it too much more work to add a test that makes an actual request?

Reply:
@lalli-flores the team discussed the integration test requirement in planning today and we have created user story SPARK-110258 to track this new requirement. Eamonn is scheduled to move on to another user story and cannot build this out further as part of his current user story.